Compilation of Quantum Hardware architectures.

shalini9894 opened the pull request:
I have compiled different qubit architecture like superconducting, silicon qubits, NV center in diamonds, Topological qubits, photonics, Ion traps, Neutral atoms. Below are the changes I made for every qubit modalities:

1. Gave a beginner friendly intro to each and every hardware modality.

2.  Included all the companies working in specific domains along with the universities and their recent breakthroughs. I also included their recent research work along with the link to every companies for all the qubit modality.

3. Included applications to each hardware modality.

4. Included diagrams or roadmap wherever necessary.

5.  Linked the educational resources for all qubit modalities.

Fixes #35

shalini9894 committed:
Quantum Hardware compilation

I have compiled different qubit architecture. Below are the changes I made:

1. Gave a beginner friendly intro to each and every hardware modality.

2.  Included all the companies working along with the universities. I also included their recent research work along with the link to  every companies for all the qubit modality

3. Included applications to each hardware modality.

4. Included diagrams or roadmap wherever necessary

5.  Linked the educational resources for all qubit modalities.

shalini9894 commented:
Hi @MaldoAlberto,
Hope you are well. I did PR for Quantum Hardware compilation. Please let me know if I need to modify anything. Thank you.

lia-approves commented:
Hello, sorry for the delay on our end and my underestimation of how much time we would need. We'll need more time to go through your pull request and discuss the decision. In any case it's safe to say we won't be considering pull requests for this Unitary Hack challenge (quantum hardwares) outside the 3 pull requests submitted so far.

shalini9894 committed:
Commit 1

shalini9894 committed:
Neutral atom qubits

shubhusion committed:
Created nmr-qubits.md

shubhusion committed:
Updated hardware pages

shubhusion committed:
image corrected

shalini9894 committed:
Modified hardware modalities

shalini9894 committed:
Merge branch 'issue-35-update-hardware' of https://github.com/shalini9894/Quantum-Universal-Education.github.io into issue-35-update-hardware

shubhusion committed:
Update ion-trap-qubits.md

shalini9894 committed:
Merge branch 'issue-35-update-hardware' of https://github.com/shalini9894/Quantum-Universal-Education.github.io into issue-35-update-hardware

NandanPatel24 committed:
Replace silicon-spin-qubits.md with updated silicon-spin.md

NandanPatel24 committed:
Rename _pages/silicon-spin-qubits.md to _pages/hardware/silicon-spin.md

NandanPatel24 committed:
Update silicon-spin.md

NandanPatel24 committed:
Update silicon-spin.md

NandanPatel24 committed:
Delete _pages/hardware/silicon-spin-qubits.md

shalini9894 committed:
Compilation of Quantum hardware architecture

1. Explained all qubit modalities with its technical details along with images.

So this PR has the following:

1. Explanation of all qubit modality.
2. Explained about key principles, their variants, decoherence and noise, applications and required images.
3. Every qubit modality has companies listed with a short description of work done along with the link to each companies.
4.  Universities working in specific modality also covered.
5. Suggested some reading materials.

Fixes #35

lia-approves commented:
Merging this pull request as the winner of Unitary Hack bounty issue #35!
Congrats to all three of you for a terrific team effort.
